tests/patient.test.js

```diff
@@ -88,4 +93,45 @@ describe('Patient', function () {
       })
     })
   })
+
+  describe.only('Patient Route', () => {
+    
+    describe('Patients', () => {
+
+      it('GET all patients', function () {
+        return agent
+        .get('/api/patient')
+        .expect(200)
+      });
+
+      it('POST one Patient', function (done) {
+        agent
+        .post('/api/patient')
+        .send({
+          'first_name': 'PatientTestF1',
+          'last_name': 'PatientTestL1',
+          'email': 'patient1@test.com',
+          'DOB': '11/18/15',
+          'gender': 'F'
+        })
+        .expect(201)
+        .end(function (err, res) {
```

Comment:
Either `end` part is not working or there is something wrong with the route. I recommend testing the route with Postman/cUrl to eliminate that possibility